apps/api/infra/api.ts

```diff
@@ -371,6 +371,8 @@ export const serviceSetup = (services: {
       LICENSE_SERVICE_BARCODE_SECRET_KEY:
         '/k8s/api/LICENSE_SERVICE_BARCODE_SECRET_KEY',
       ULTRAVIOLET_RADIATION_API_KEY: '/k8s/api/ULTRAVIOLET_RADIATION_API_KEY',
+      UMBODSMADUR_SKULDARA_COST_OF_LIVING_CALCULATOR:
```

Comment:
~This variable name could be more descriptive of what it is storing, without context it is hard to tell.

Reply:
Agreed, just by adding the postfix "_URL" would be more clear since that's what it's storing.

@mannipje cc